libipl/p10/ipl0.C

```diff
@@ -605,9 +618,15 @@ static int ipl_sbe_start(void)
 		}
 	}
 
-	if (!ipl_check_functional_master()) {
-		ipl_error_callback(IPL_ERR_PRI_PROC_NON_FUNC);
-		return 1;
+	if(!rc) {
```

Comment:
is this condition correct? at line 609 we set to booted and make rc=0 and here we are asking for check_cfam?

Reply:
yes , default value of rc = 1 (error) , rc = 0 mean we should get updated the states.
Fixed another issue in rc handling in mpipl path now.